MultiEditUtils.py

```diff
@@ -4,15 +4,37 @@
 
 class MultiFindAllCommand(sublime_plugin.TextCommand):
 
-  def run(self, edit):
+  def run(self, edit, case=True, word=False, ignore_comments=False):
 
     view = self.view
     newRegions = []
+    selected_words = [view.substr(view.word(sel)).lower() for sel in view.sel()]
 
     for region in view.sel():
       substr = view.substr(region)
-      for region in view.find_all(substr, sublime.LITERAL):
-        newRegions.append(region)
+
+      if case:
+          for region in view.find_all(substr, sublime.LITERAL):
+            newRegions.append(region)
+      else:
+          for region in view.find_all(substr, sublime.LITERAL | sublime.IGNORECASE):
+            newRegions.append(region)
+
+      if word:
+        deleted = []

```

Comment:
Can you extract Lines 24 to 27 into an inline function which takes a region as a parameter and returns a boolean indicating whether the region should be ignored or not. Then, we do not need the `deleted` list variable and could call said function directly in the for comprehension in Line 28.